helpers/addresses.py

```diff
@@ -214,6 +214,7 @@
         "bFpMbtcHbtc": "0x26B8efa69603537AC8ab55768b6740b67664D518",
         "bbveCVX-CVX-f": "0x937B8E917d0F36eDEBBA8E459C5FB16F3b315551",
         "remBADGER": "0x6aF7377b5009d7d154F36FE9e235aE1DA27Aea22",
+        "remDIGG": "0x8B2a18b6400338272FDD1B991F5163E21723AF60",
```

Comment:
@sajanrajdev scripts looks good and cli commands exec smoothly

just having a doubt on the sett address based on this [comment]( https://github.com/Badger-Finance/badger-strategies/issues/35#issuecomment-1026838777)

which is the right address 0x99F39D495C6A5237f43602f3Ab5F49786E46c9B0 VS 0x8B2a18b6400338272FDD1B991F5163E21723AF60?

Reply:
Yes Petro, a new vault was redeployed this morning and I think Alex edited the comment there. Aas about to get to change it. Thanks for checking!